Update index.md

akadidas opened the pull request:
Some much needed change in language and ambiguos defination and examples.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

akadidas committed:
Update index.md

Some much needed change in language and ambiguos defination and examples.

RandellDawson commented:
Please use PR titles which explain what you are changing.  If it is a translation, please also indicate **translation: language name**. This let's the reviewer quickly know what kind of contribution you have made.

Also, please read/check all 5 items above indicating you have fulfilled each requirement.

akadidas committed:
Required_changes.md

Several changes are made in definations along with some easily understoodable examples . Previously the article have some use of ambiguous language .Now those are removed or improvised

akadidas committed:
Merge pull request #1 from akadidas/akadidas-patch-1

Required_changes.md

RandellDawson reviewed (changes requested):
Since you deleted the two examples, can you provide some new examples?

Thank you.
